apps/vs-code-react/src/app/dataMapper/appV2.tsx

```diff
@@ -0,0 +1,216 @@
+import { changeFetchedFunctions, changeSourceSchema, changeTargetSchema } from '../../state/DataMapSlice';
+import type { AppDispatch, RootState } from '../../state/store';
+import { VSCodeContext } from '../../webviewCommunication';
+import {
+  getFileNameAndPath,
+  DataMapDataProvider,
+  DataMapperDesigner,
+  DataMapperDesignerProvider,
+  InitDataMapperApiService,
+  defaultDataMapperApiServiceOptions,
+  getFunctions,
+  getSelectedSchema,
+} from '@microsoft/logic-apps-data-mapper-v2';
+import { getTheme, useThemeObserver } from '@microsoft/logic-apps-designer';
+import type { Theme } from '@microsoft/logic-apps-shared';
+import type { MessageToVsix } from '@microsoft/vscode-extension-logic-apps';
+import { ExtensionCommand } from '@microsoft/vscode-extension-logic-apps';
+import { useCallback, useContext, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+export const DataMapperAppV2 = () => {
```

Comment:
For DataMapper V2, can we just have an empty canvas for now? Things will change with v2 so I guess we don't really need all of the same logic as v1